.github/scripts/check-engines-cli.sh

```diff
@@ -52,36 +52,32 @@ esac
 echo "Assumed OS: $os_name"
 echo "CLI_QUERY_ENGINE_TYPE == $CLI_QUERY_ENGINE_TYPE"
 
+ENGINES_PACKAGE=$(node -e "console.log(path.dirname(require.resolve('@prisma/engines/package.json', {paths: [path.dirname(require.resolve('prisma/package.json'))]})))")
+
 if [ $CLI_QUERY_ENGINE_TYPE == "binary" ]; then
   echo "Binary: Enabled"
   case $os_name in
     linux)
-      qe_location="node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x"
-      qe_location2="node_modules/prisma/node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x"
+      qe_location="$ENGINES_PACKAGE/query-engine-debian-openssl-1.1.x"
```

Comment:
These are two different locations that are usually both with a QE. Why only check one now?

Reply:
I thought the one was a fallback because `@prisma/engines` is always a dependency of `prisma` and is never installed on it's own. For that reason, I opted for looking it up via it's parent dependency instead of via it's potential direct path.

Reply:
Probably correct. Line 107 can be updated as well then to not check the second path any more.

Reply:
(Although I wonder a bit why the fallback was necessary... if it was.)

Reply:
I think the fallback would be useful for different package managers, as they nest and hoist packages differently. This version  for finding the package is more universal. I will remove the forgotten check in the if.